templates/account/signup.html

```diff
@@ -52,8 +44,8 @@ <h3 class="welcome">Welcome!</h3>
                 </div>
             </form>
         </div>
-        <div class="h-100 hidden-sm col-xl-4 img-block mw-100"
-            style="background-image : url('{% static "mainapp/png/BookSignIn.png" %}')">
+        <div class="h-100 hidden-sm col-xl-4 img-block mw-100" style="background-image : url('{% static "
```

Comment:
The template tags should not be continued on new line, I think that's whats causing the issue.

Reply:
The tags are coming on a new line because of in built formatting in VS code.
I do not think that's what causing the issue.

Reply:
Could you confirm and update with the latest ss? Because django sees it as a placeholder and places the url there.

Reply:
Yes I just confirmed and it is indeed the formatting. No matter how many times I take it on a single line, it still comes on a next line after saving.

Reply:
Hi @Abg4real it's because your vscode is formatting the file as per HTML syntax.
You can follow these steps :
1.At bottom when you open sigup.html you can see file formatting is set to Html.
![1st](https://user-images.githubusercontent.com/57425995/105069533-1a77da80-5aa8-11eb-9c1a-047266f8e095.png)
2.Add django extension to vscode (in extension section search for django template )
![2nd](https://user-images.githubusercontent.com/57425995/105069713-527f1d80-5aa8-11eb-8cb5-471b1a7dee83.png)
3.the signup.html file will now be recognized as django template you can even change it by yourself to Django template.
![4](https://user-images.githubusercontent.com/57425995/105070471-58c1c980-5aa9-11eb-8728-5284743d91cf.png)

Hope this helps.

Reply:
Hi @maneprajakta 
I followed your steps and I was able to get rid of the error in vscode. 
Now, the lines are being formatted properly.
![image](https://user-images.githubusercontent.com/55308761/105279290-43de5680-5bcd-11eb-8fc7-153d861bb16a.png)
But, the problem still persists.
![image](https://user-images.githubusercontent.com/55308761/105279642-029a7680-5bce-11eb-93da-012dfce457b5.png)
I tried clearing browser cache too.
Do you think the local server might be creating some cache which I might need to clean to see the changes take effect?

Reply:
Try updating your branch with remote

Reply:
@Abg4real the ss attached is of sign-in and signup button in login.html page can you check for this one in login.html
`  <a class="forgotpassword" href="{% url 'account_reset_password' %}">{% trans "Forgot Password?" %}</a>
 `
it should be in one line.